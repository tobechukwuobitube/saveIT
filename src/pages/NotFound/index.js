import React, { Component } from 'react';
import '../../assets/css/createAccount.css';

export default class NotFound extends Component {
  render() {
    return (
      <div>
        <h1 className="noParcels">404 Not Found</h1>
      </div>
    );
  }
}