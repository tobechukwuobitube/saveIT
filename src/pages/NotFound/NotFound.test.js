import React from 'react';
import renderer from 'react-test-renderer';
import NotFound from './';

test('NotFound snapshot test', () => {
  const wrapper = renderer.create(<NotFound />);
  expect(wrapper).toMatchSnapshot();
});
