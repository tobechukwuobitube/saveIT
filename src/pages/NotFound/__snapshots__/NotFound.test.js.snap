// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound snapshot test 1`] = `
<div>
  <h1
    className="noParcels"
  >
    404 Not Found
  </h1>
</div>
`;
